makefiles/tools/riotboot_serial.inc.mk

```diff
@@ -0,0 +1,13 @@
+RIOTBOOT_SERIAL := $(RIOTTOOLS)/riotboot_serial/riotboot_serial
+
+FLASHER   ?= $(RIOTBOOT_SERIAL)
+FLASHFILE ?= $(HEXFILE)
+PROG_BAUD ?= 115200
```

Comment:
You could just reuse BAUD instead of redefining a new variable.

Reply:
I copied that form `cc2538-bsl` - it's independent from the application's serial baud rate